_dockerPublisher/Dockerfile

```diff
@@ -24,14 +24,20 @@ RUN dnf install -y rsync
 RUN dnf install -y openssh-clients
 RUN dnf install -y net-tools
 RUN dnf install -y iputils
+RUN dnf install -y wget
```

Comment:
@mbiarnes just a generic comment regarding dockerfiles:
Every RUN command in dockerfile creates separate filesystem layer. Having too many layers can lead to problems (not sure, but I think performance of the filesystem). There is general advice to try to [minimize number of layers](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers) in the dockerfile.

In this case you can "squash" all the `dnf install` commands into one RUN like so:

`RUN dnf install -y ruby rubygems rubygem-rake rubygem-bundler ruby-devel rpm-build gcc-c++ make rsync openssh-clients net-tools iputils wget`

In other cases you can reduce the number of RUNs like by using 
```
RUN command1 && command2 && command3
```
instead of 
```
RUN command1
RUN command3
RUN command3
```

Reply:
when I have time I will look at this